src/data/metadata.csv

```diff
@@ -13246,6 +13250,7 @@ IN_TN_MDR,IN,India,TN,Tamil Nadu,MDR,Madurai,,,,
 IN_TN_NIL,IN,India,TN,Tamil Nadu,NIL,Nilgiris,,,The Nilgiris,
 IN_TN_NMK,IN,India,TN,Tamil Nadu,NMK,Namakkal,,,,
 IN_TN_NPM,IN,India,TN,Tamil Nadu,NPM,Nagapattinam,,,,
+IN_TN_MDT,IN,India,TN,Tamil Nadu,MDT,Mayiladuthurai,,,,
```

Comment:
This should probably be moved up. From the unit test error logs: `AssertionError: 'IN-TN-MDT' not greater than 'IN-TN-NPM' : Keys in metadata.csv must follow lexicographical order: IN-TN-MDT ≤ IN-TN-NPM`

Reply:
Ah thank you!

Reply:
Done.